src/assets/custom/css/_sass/_services-curated-publications.scss

```
@@ -0,0 +1,16 @@
+.services__curated-publications {
+  @include small {
+    .page-location-anchor {
+      top: calc(0px - #{$site-navbar-height-base});
+    }
+  }
+}
+
+.services__curated-publications--one-publication {
+   margin-top: -150px;
+   @include small {
+    .page-location-anchor {
+      top: calc(0px - #{$site-navbar-height-base});
```

I think this would work just as well 

```suggestion
      top: -$site-navbar-height-base;
```

---

Same on line 4

---

This is solved within [this PR](https://github.com/codurance/site/pull/1819).